wfl/Database/MongoDBManager.cs

```diff
@@ -19,52 +19,60 @@ public MongoDBManager()
             db = client.GetDatabase("WFL");
         }
 
-        public async Task InsertRestaurantAsync(RestaurantDto r)
+        public async Task<Restaurant> InsertRestaurantAsync(RestaurantDto r)
         {
-            IMongoCollection<RestaurantDto> collection = db.GetCollection<RestaurantDto>("restaurants");
-            await collection.InsertOneAsync(r);
+            IMongoCollection<Restaurant> collection = db.GetCollection<Restaurant>("restaurants");
+            Restaurant restaurant = new Restaurant()
+            {
+                Name = r.Name
+            };
+            await collection.InsertOneAsync(restaurant);
+            return restaurant;
         }
 
-        public async Task<RestaurantDto> UpdateRestaurantAsync(string id, RestaurantDto restaurant)
+        public async Task<Restaurant> UpdateRestaurantAsync(string id, RestaurantDto r)
         {
-            IMongoCollection<RestaurantDto> collection = db.GetCollection<RestaurantDto>("restaurants");
-            var filter = Builders<RestaurantDto>.Filter.Eq("Id", id);
+            IMongoCollection<Restaurant> collection = db.GetCollection<Restaurant>("restaurants");
+            var filter = Builders<Restaurant>.Filter.Eq("Id", id);
+            Restaurant restaurant = new Restaurant
+            {
+                Name = r.Name
```

Comment:
Do we need to also update the MenuLink here? Would this effectively wipe out the MenuLink?